docs/pages/installation/codespaces.md

```diff
@@ -0,0 +1,104 @@
+
+**To Setup and run Meshery on Minikube** :
+
+- [Steps](#steps)
+  - [1. Start minikube](#1-start-minikube)
+  - [2. Install Meshery](#2-install-meshery)
+  - [3. Configure Meshery to use minikube](#3-configure-meshery-to-use-minikube)
+  - [Manual Steps](#manual-steps)
+
+## Steps
+
+After starting the Codespace on the Meshery repo, perform the following steps in the `workspaces` folder in order to run Meshery on Minikube using GitHub Codespaces:
+
+### 1. Start minikube
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ minikube start --cpus 4 --memory 8192 --kubernetes-version=v1.16.1
+ </div></div>
+ </pre>
+
+_Note: minimum memory required is --memory=4096 (for Istio deployments only)_
+
+**Check up on your minikube cluster** :
+
+<pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+  minikube status 
+ </div></div>
+ </pre>
+
+### 2. Install Meshery
+
+<pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+  $ curl -L https://meshery.io/install | PLATFORM=docker bash - 
+ </div></div>
+ </pre>
+<!-- Follow the [installation steps](/installation/quick-start) to setup the mesheryctl CLI and install Meshery. -->
+**Users using docker driver**:
+After completing the Meshery installation, execute the following commands to establish connectivity between Meshery Server and Kubernetes cluster:
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ docker network connect bridge meshery_meshery_1
+ </div></div>
+ </pre>
+
+<br/>
+
+<pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ docker network connect minikube meshery_meshery_1
+ </div></div>
+ </pre>
+
+To establish connectivity between a particular Meshery Adapter and Kubernetes server, use _"docker ps"_ to identify the name of the desired container, and execute the following commands:
+
+<pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ docker network connect bridge &#60; container name of the desired adapter &#62;
+ </div></div>
+ </pre>
+
+<br/>
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ docker network connect minikube &#60; container name of the desired adapter &#62;
+ </div></div>
+ </pre>
+
+### 3. Configure Meshery to use minikube
+
+1. Login to Meshery which is running on a port dynamically generated by Codespace.
+
+2. To allow Meshery to auto detect your config file, Run :
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ kubectl config view --minify --flatten > config_minikube.yaml
+ </div></div>
+</pre>
+
+<br />
+Meshery should now be connected with your managed Kubernetes instance. Take a look at the [Meshery guides]({{ site.baseurl }}/guides) for advanced usage tips.
+
+<br />
+#### Note: Run Codespaces on locally installed VSCode IDE for a better experience.
```

Comment:
You could include the instructions to access CodeSpaces VM in VS Code here. It's just feeling blank here

Reply:
Sure, thanks. I have updated the doc with an inline link describing how one can open Codespace in the locally installed VS Code along with other options.